server.js

```diff
@@ -1,5 +1,5 @@
 const app = require('./api/app.js');
 
-const port = process.env.PORT || 8000;
+const port = process.env.PORT || 8080
```

Comment:
So this is how you leave a comment on a particular part of the code!  This is where it shows a missing semicolon. :)

Reply:
I took care of it.